src/components/SceneConfigMenu.js

```diff
@@ -263,7 +263,10 @@ class ConfigModal extends Component {
     return (
       <ButtonBase
         style={style}
-        onClick={() => this.props.sceneActions.toggleCoordSky()} >
+        onClick={() => {
+          this.props.handleRender();
```

Comment:
Can we include this logic in all of the menu options? Sky color, floor color, hiding the editor, and removing the floor all also have the same issue as #89.

Reply:
my last commit added this logic for changes to sky/floor color and removing the floor, but not for hiding the editor, as all of the other settings require the scene to be re-rendered anyway. i don't believe hiding the editor does, and i believe @jSwanson99's fix to toggling the editor not saving works better for this specific case.